std_msgs/Header header

geometry_msgs/Point[] center_points
geometry_msgs/Point[] inner_points
geometry_msgs/Point[] outer_points
